examples/Demo/FluentUI.Demo.Client/Documentation/Components/Slider/FluentSlider.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Slider
+route: /Slider
+---
+
+# Slider
+
+The **FluentSlider** component is an input where the user selects a value from within a given range. Sliders typically have a slider thumb that can be moved along a bar, rail, or track to change the value of the slider.
+
+Default settings are `Min=0`, `Max=100` and `Step=1`.
+
+`TValue` can be one of the following `Type`'s:  
+
+- `byte`
+- `int`  
+- `long`  
+- `short`  
+- `float`  
+- `double`  
+- `decimal`  
+
+View the [Usage Guidance](https://fluent2.microsoft.design/components/web/react/slider/usage).
+
+{{ SliderDefault }}
+
+## Label
+
+{{ SliderLabel }}
+
+## Orientation
+
+{{ SliderOrientation }}
+
+## Size
+
+{{ SliderSizes }}
+
+## Thumbs
+
```

Comment:
Can you explain what `slot=“ thumb”` is and when it might be interesting to use it?